src/javascript/js/bridge.js

```diff
@@ -252,6 +257,11 @@ const ipc = {
     debug('js -> py', data)
     process.stderr.write(JSON.stringify(data) + '\n')
   },
+  sendBlob: (data, r) => {
+    process.stderr.write('blob!{"r":'+r+',"len":'+data.length+'}!')
```

Comment:
Some future ideas:
* To make this more universal, we could allow the caller to pass in length explicitly instead of always requiring a `length` attribute (though it could still be the fallback).
* One could also consider adding a way to retrieve only a specific segment of the data. On the other, it should be possible to extract the segment as an own JS object first and transfer that instead, so perhaps this is not worth adding.

WDYT?

Reply:
I'll postpone caller-provided length to a future consideration/PR. I suppose it could be a bit problematic since a bad value might break the IPC.